libraries/SrcWrapper/src/HAL/stm32yyxx_hal.c

```diff
@@ -26,6 +26,8 @@
   #include "stm32l5xx_hal.c"
 #elif STM32MP1xx
   #include "stm32mp1xx_hal.c"
-#elif STM32WBxx
-  #include "stm32wbxx_hal.c"
+#elif STM32BLxx
+  #include "stm32blxx_hal.c"
```

Comment:
```suggestion
#elif STM32WBxx
  #include "stm32wbxx_hal.c"
```

That's why I would add the WL using the script to avoid such issue.

Reply:
> That's why I would add the WL using the script to avoid such issue.

sorry, where can I find the script?

Reply:
I'm currently updating it to enhance STM32 series addition.
I will provide you tomorrow the WL Cube drivers introduction.